src/test/java/uk/gov/pay/products/validations/PaymentRequestValidatorTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.pay.products.validations;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.products.util.Errors;
+
+import java.util.Optional;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class PaymentRequestValidatorTest {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private PaymentRequestValidator requestValidator;
+
+    @Before
+    public void before() throws Exception {
+        requestValidator = new PaymentRequestValidator(new RequestValidations());
+    }
+
+    @Test
+    public void shouldSuccess_onCreatePayment_ifJsonPayloadIsNull() throws Exception {
+        Optional<Errors> errors = requestValidator.validatePriceOverrideRequest(null);
+        assertFalse(errors.isPresent());
```

Comment:
assertThat(..., is()); perhaps?

Reply:
not sure we actually standardise on hamcrest. I don't think we should. at least on these kind of simple assertions the standard Junit ones are perfectly readable to me. :|